tldr.py

```diff
@@ -504,6 +508,40 @@ def main() -> None:
                 with open(command, encoding='utf-8') as open_file:
                     output(open_file.read().encode('utf-8').splitlines(),
                            plain=options.markdown)
+    elif options.search:
+        command = '-'.join(options.command)
+        page = None
+        maxprob = 0
+        searchquery = options.search.split(' ')
+
+        for i in get_commands():
+            if i.startswith(command):
+                for p in get_platform_list():
```

Comment:
Should store the result of `get_platform_list` instead of calling it repeatedly during execution.

```suggestion
        platforms = get_platform_list()
        for i in get_commands(platforms):
            if i.startswith(command):
                for p in platforms:
```